### DISCUSSION TITLE
How to apply the trained MMM model to new data to predict

### DISCUSSION THREAD

**Discussion by yongqiangzhangzazzle** [google/meridian#214] [Q&A]
Meridian supports prediction function: say we split the data into training data (01/01/2021 ~ 03/31/2024) and the test data (04/01/2024~06/30/2024), and we train the model with training data and apply the model to the testing data and measure the accuracy. Is it possible to do prediction with Meridian? Thanks!

**Comment by mikewurm** [comment]
Meridian is intended for causal inference, not prediction. Trend and seasonality effects, estimated via the knot parameters, are estimated within only the model training window.

More information can be found in the documentation on [holdout observations](https://developers.google.com/meridian/docs/basics/holdout-observations?hl=en) and [assessing model fit](https://developers.google.com/meridian/docs/advanced-modeling/model-fit?hl=en).

[Discussion thread #197](https://github.com/google/meridian/discussions/197) may also be relevant.

**Comment by hdesgran** [comment]
This feature would be highly appreciated indeed.
Without a predict function, it's challenging to implement and leverage Meridian for practical, day-to-day use.
